.github/workflows/Dockerfile.nginx

```diff
@@ -2,6 +2,10 @@
 # built in the host in order to be created and deployed
 
 FROM nginx:alpine
+
+LABEL maintainer "Jellyfin Packaging Team - packaging@jellyfin.org"
+LABEL org.opencontainers.image.source https://github.com/jellyfin/jellyfin-vue
```

Comment:
```
LABEL maintainer "Jellyfin Packaging Team - packaging@jellyfin.org" \
    org.opencontainers.image.source https://github.com/jellyfin/jellyfin-vue
```

Reply:
fair enough, was just a suggestion since I find this to be cleaner than multiple label instructions. Opinions differ and that is a good thing, I am fine either way :wink:

Reply:
I'll keep them in separate label statements as is what the Dockerfile reference suggests as [best practice](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#label)

Reply:
well the best practices list both ways, either is fine. The single instruction originated from the legacy "issue" of one layer per `LABEL` which has been fixed long ago, however I still prefer it over multiplying the instruction (personal preference).